packages/web-react/src/components/Collapse/demo/index.tsx

```diff
@@ -4,16 +4,16 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 // eslint-disable-next-line @typescript-eslint/ban-ts-comment, import/extensions, import/no-unresolved
 // @ts-ignore: No declaration file
```

Comment:
Is this comment still relevant? If not, we can happily delete it. If yes, a reason would be helpful.

Reply:
This is related to the https://jira.almacareer.tech/browse/DS-561. Icons are not exporting the declaration files for Typescript usage.

Reply:
I have modified the comment in all places. Thanks.